chapters/solutions.Rmd

```diff
@@ -677,15 +691,25 @@ ENIAC was the world's first general-purpose electronic computer.
 
 ### Exercise \@ref(git-cmdline-ex-diff) {-}
 
-FIXME: solution for exercise on git diff
+1. `git diff HEAD~9 bin/plotcounts.py` compares what has changed between the 
```

Comment:
I don't really understand the exercise associated with this solution, which currently reads as follows:
1.   What will the command `git diff HEAD~9 bin/plotcounts.py` do if we run it?
2.  What does it actually do?

What is the difference between "what the command does if we run it" and "what it actually does"? It's basically asking the same thing twice.

Also, we might want to use `HEAD~1` instead of `HEAD~9`. At this point of the book there have been only four commits to the repo, and only two of them (the initial commit and then one update to the script) involve `plotcounts.py`.

Reply:
:+1: to your suggestions - the question was originally "What do you think it will do?" and "What does it actually do?" The trick is that there isn't enough history for `HEAD~9` to be meaningful because (as you point out) there have only been four commits.